runner/src/ipython.rs

```diff
@@ -0,0 +1,34 @@
+use pyo3::prelude::*;
+
+#[pyclass]
+struct IPython;
+
+#[pymethods]
+impl IPython {
+    fn system(&self, py: Python<'_>, code: &str) -> PyResult<()> {
```

Comment:
Do you think it's possible to suspend the progress bar when this function is called? So that `!echo foobar` prints

Reply:
i'll give it a shot (: